scripts/ci-e2e.sh

```diff
@@ -81,10 +49,19 @@ source "${REPO_ROOT}/hack/ensure-kubectl.sh"
 # shellcheck source=./hack/ensure-kustomize.sh
 source "${REPO_ROOT}/hack/ensure-kustomize.sh"
 
-# This will run the tests with env variabls defined in environment.sh
-# or exported by metal3-dev-env scripts
-${M3_DEV_ENV_PATH}/scripts/run_command.sh make e2e-tests
+# shellcheck source=./hack/e2e/images.sh
+source "${REPO_ROOT}/hack/e2e/images.sh"
+# shellcheck source=./hack/e2e/releases.sh
+source "${REPO_ROOT}/hack/e2e/releases.sh"
+# shellcheck source=./hack/e2e/ironic_basic_auth.sh
+source "${REPO_ROOT}/hack/e2e/ironic_basic_auth.sh"
+# shellcheck source=./hack/e2e/ironic_tls_setup.sh
+source "${REPO_ROOT}/hack/e2e/ironic_tls_setup.sh"
```

Comment:
Because cloning metal3-dev-env is mandatory for now, these scripts should be sourced from the repo. Keeping track of two copies of the same scripts in two different repos makes it challenging to maintain e2e tests.